src/settings/Settings.js

```diff
@@ -275,6 +275,11 @@ export const SETTINGS = {
         supportedLevels: LEVELS_ACCOUNT_SETTINGS,
         default: [],
     },
+    "use_system_theme": {
+        supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS,
+        default: true,
+        displayName: _td("Match system theme"),
```

Comment:
Wonder if we should make the label more obvious that this is about dark mode rather than the system theme. E.g. doesn't use system theme colors, ... Perhaps "Match system dark mode setting"?

Reply:
Yeah, that might be clearer